test/spec/common/index/exportRules.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
```

Comment:
Why not just a test-case/ https://github.com/algolia/algoliasearch-client-javascript/tree/develop/test/spec/common/client/test-cases?

Reply:
Because it's not a single API call, it's more like `browseAll`

Reply:
got it